### PR TITLE
initial-scaffolding

### PR DESCRIPTION
**Pull request by unawaretub86**

### CODE REVIEW

File: internal/domain/money-tracker/repository/database/yourDomain.go [hunk 1]
```diff
@@ -0,0 +1 @@
+package database
```
Comment (yipson) [inline internal/domain/money-tracker/repository/database/yourDomain.go]: please review

File: internal/domain/money-tracker/repository/yourDomain.go [hunk 2]
```diff
@@ -0,0 +1 @@
+package repository
```
Comment (yipson) [inline internal/domain/money-tracker/repository/yourDomain.go]: please review

**Review by yipson — CHANGES_REQUESTED** [review]
please review the database folder

**Comment by unawaretub86** [comment]
localhost:8080/ping

{
    "response": "pong"
}

image

**Review by yipson — APPROVED** [review]

**Pull request merged**